test/drawers/drawerTests.ts

```diff
@@ -22,17 +21,17 @@ class MockAnimator implements Plottable.Animator {
 }
 
 function createMockDrawer(dataset: Plottable.Dataset) {
-  var drawer = new Plottable.Drawer(dataset);
+  const drawer = new Plottable.Drawer(dataset);
   (<any> drawer)._svgElementName = "circle";
   return drawer;
 }
 
 describe("Drawers", () => {
   describe("Abstract Drawer", () => {
-    var oldTimeout: any;
-    var timings: number[] = [];
-    var svg: d3.Selection<void>;
-    var drawer: Plottable.Drawer;
+    let oldTimeout: any;
+    let timings: number[] = [];
+    let svg: d3.Selection<void>;
+    let drawer: Plottable.Drawer;
```

Comment:
It's declared as `let` here because we do not know the value of this variable (it is not initialized to a value here)

The same variable name is declared as a `const` above because these two variables are fundamentally different variables.  The `drawer` variable in `createMockDrawer()` is used to hold a creation of a new mock drawer.  However, the `drawer` variable in this describe suite is used to switch between creations of new drawers to use across unit tests.